hack/move-upstream.sh

```diff
@@ -18,6 +18,8 @@ repo="${UPSTREAM_REPO:-k8s.io/kubernetes}"
 package="${UPSTREAM_PACKAGE:-pkg/api}"
 
 patch="${TMPDIR}/patch"
+rm -rf "${patch}"
+mkdir -p "${patch}"
 relativedir="../../../${repo}"
```

Comment:
Because you can't easily solve absolute paths across Linux Mac and it wasn't worth it to me.